myproject/supermarket_manager/urls.py

```diff
@@ -26,5 +28,6 @@
     path("api/accounts", add_account, name="add-account"),
     path("api/accounts/<str:account_id>", get_account_detail, name="get-account-detail"),
     path("api/logins", login_account, name="login-account"),
-    path("api/accounts/<str:account_id>", update_account, name="update-account"),
+    path("api/accounts/<str:account_id>", change_password, name="update-account"),
+    path("api/logouts", logout_account, name="logout-account")
```

Comment:
Similar : `path("api/logouts", logout_account, name="logout-account")`  --> `path("api/logouts", logout, name="logout")`

Please create new Github issue to fix the API endpoints.